blueocean-pipeline-scm-api/src/main/java/io/jenkins/blueocean/scm/api/AbstractPipelineCreateRequest.java

```diff
@@ -52,6 +55,16 @@ public AbstractPipelineCreateRequest(String name, BlueScmConfig scmConfig) {
     }
 
     protected ModifiableTopLevelItemGroup getParent() {
-        return Jenkins.getInstance();
+        String organization = getOrganization();
+        ItemGroup parent =  OrganizationResolver.getItemGroup(getOrganization());
+        if(parent == null){
+            throw new ServiceException.BadRequestExpception("Invalid Jenkins organization. " + organization + ". Not found");
+        }
+
+        if(!(parent instanceof ModifiableTopLevelItemGroup)){
```

Comment:
@vivek I think we should refactor `OrganizationResolver.getItemGroup` to return `ModifiableTopLevelItemGroup` rather than `ItemGroup` as `Folder`, `Jenkins` and `Hudson` are the only possible item groups that could be orgs and can have `TopLevelItem`s created in them with `createProject` 🐜